server/src/api/contacts/contacts.service.ts

```diff
@@ -49,7 +49,7 @@ export const findByName = async (firstname: string, lastname: string): Promise<r
 
 export const find = async (id: string): Promise<response> => {
   const myQuery = {
-    text: 'SELECT * FROM contacts WHERE customerid = $1',
+    text: 'SELECT * FROM contacts WHERE contactid = $1',
```

Comment:
Works without the semi-colon but should add it anyways for consistency

Reply:
Fixed in next push